spec/lib/services/nodes-api-service-spec.js

```diff
@@ -499,13 +506,24 @@ describe("Http.Services.Api.Nodes", function () {
             .to.equal(undefined);
         });
 
-        it("_removeRelation should fail if don't have targets", function() {
+        it("_removeRelation should return undefined if don't have relations", function() {
+            var enclNode = {
+                    id: '1234abcd1234abcd1234abcd',
+                    type: 'enclosure'
+                };
+
+            expect(nodeApiService._removeRelation(enclNode, 'encloses', computeNode))
```

Comment:
The function that begins with `_` is considered as private function, in unit-test, it better to focus on test those public function (or interfaces). The private function is purely the design internal.

This will leave us the flexibility to refactor the code in future (keep interface unmodified, but change implementation)